src/cart/index.html

```diff
@@ -8,26 +8,29 @@
 
     <title>Sleep Outside | Cart</title>
 
-    <link rel="stylesheet" href="/css/style.css" />
+    <link rel="stylesheet" href="../css/style.css" />
 
     <script src="../js/cart.js" type="module"></script>
-  </head>
+     </head>
 
   <body>
-    <header class="divider" id="main-header"></header>
+    <header class="divider" id="main-header">
+      
+    </header>
 
     <main class="divider">
       <section class="products">
         <h2>My Cart</h2>
 
-        <ul class="product-list"></ul>
-        <div class="list-footer hide">
-          <a href="/checkout/index.html">Checkout</a>
-          <p class="list-total">Total:</p>
-        </div>
+        <ul class="product-list">
+        </ul>
       </section>
+      <button id="checkout-button"><a href="../checkout/index.html">Checkout</a></button>
+      <div class="cart-footer hide">
+        <p class="cart-total">Total:</p>
```

Comment:
In your JavaScript these are called `list-footer` and `list-total` instead of `cart-*`. I got an error because of that.